apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,58 @@
-import { Component, OnInit } from '@angular/core';
+import { NgOptimizedImage } from '@angular/common';
+import { Component, OnInit, inject } from '@angular/core';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { City } from '../../model/city.model';
+import { CardComponent } from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
+import { NgTemplateListItemDirective } from '../../ui/list-item/ng-template-list-item.directive';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card
+      [list]="cities"
+      [itemTemplate]="itemTemplate"
+      backgroundClass="bg-purple-200"
+      (clickedAdd)="addCity()">
+      <img ngSrc="assets/img/city.png" width="200" height="200" priority />
+    </app-card>
+
+    <ng-template listItem [infer]="cities" #itemTemplate let-item>
+      <app-list-item
+        [name]="item.name"
+        [id]="item.id"
+        (clickedRemove)="removeCity($event)" />
+    </ng-template>
+  `,
   standalone: true,
-  imports: [],
+  imports: [
+    NgOptimizedImage,
+    CardComponent,
+    ListItemComponent,
+    NgTemplateListItemDirective,
+  ],
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  #http = inject(FakeHttpService);
+  #store = inject(CityStore);
+
+  cities: City[] = [];
+
+  ngOnInit(): void {
+    this.#http.fetchCities$.subscribe((cities) => this.#store.addAll(cities));
+
+    this.#store.cities$.subscribe((cities) => (this.cities = cities));
```

Comment:
Definetly right @svenson95, thanks for mentioning it.

I focused on the content projection part because I saw the next challenge is about refactoring a CRUD-Application.